chukwuemekachm/utility_manager_web
# Chima: Setup linting with eslint and prettier

#### What does this PR achieve?
- This PR introduces the required changes to setup linting on the project with eslint and prettier. It also lints the current files and resolves all linting issues.

#### Any background context?
- N/A

#### Manual Testing Steps
- Run `yarn` to install dependencies
- Run `yarn lint` to test linting and view results
- Make any commit and see the commit hooks in action

#### Screenshots:
- N/A

#### Relevant Trello Tickets
- [Ticket #42](https://trello.com/c/mQWPMBEA)
